tests/Kiota.Builder.Tests/KiotaBuilderTests.cs

```diff
@@ -184,6 +184,7 @@ public async Task ParsesEnumDescriptions()
         - Standard_GRS
         - Standard_RAGRS
         - Premium_LRS
+        - Premium_LRS
```

Comment:
I should have added the failing scenario to the issue. These two sets of enum values were being generated with the same name:
- values `Etc/GMT0` and `Etc/GMT-0` both generated as `ETC_G_M_T0 = 'Etc/GMT-0'`
- values `GMT0` and `GMT-0` both generated as `G_M_T0 = 'GMT-0'`

Current fix causes one of the enum values to not be generated.

Reply:
This suggests an issue/change with the way special characters are stripped from the description vs the issue of duplicate Enums being generated in the code. 
I think this should be followed up via https://github.com/microsoft/kiota/issues/2495 as this would probably mean a breaking change in Kiota as the issue suggests.
What do you think?

Reply:
your comments pretty much sums it all. I'd rather be missing one entry and a result that builds at this point than having twice the same entry (still missing one) and something that doesn't build. Since fixing the normalization logic would be a breaking change, we'd need to major rev it. We're thinking about getting to a v2 faster to address all those misses.

Reply:
Thanks for clarifying. This now makes sense. Wasn't aware of the breaking change.